### DISCUSSION TITLE
How to change the error message when an endpoint errors?

### DISCUSSION THREAD

**Discussion by TechplexEngineer** [sveltejs/kit#5553] [Q&A]
For example if I have an endpoint with this get handler:
```typescript
export const get: RequestHandler = async () => {
	return {
		status: 400,
		body: new Error("Not yet implemented")
	};
};
```

The frontend shows a stack trace prefixed with "Failed to load data". How can I change the error message?

**Comment by mustofa-id** [comment]
Can't see this in [latest version](https://stackblitz.com/edit/sveltejs-kit-template-default-aeyzbh?file=src%2Froutes%2Findex.js&terminal=dev) of Kit.
